plugins/uefi/README.md

```diff
@@ -25,18 +16,15 @@ UEFI Unlock Support
 
 On some Dell systems it is possible to turn on and off UEFI capsule
 support from within the BIOS.  This functionality can also be adjusted
-from within the OS by fwupd. This requires using fwupdate 5 or later
-and compiling it with libsmbios support.
+from within the OS by fwupd. This requires compiling with libsmbios support.
 
-When fwupd and fwupdate have been compiled with this support you will
-be able to enable UEFI support on the device by using the `unlock` command.
+When fwupd has been compiled with this support you will be able to enable UEFI
+support on the device by using the `unlock` command.
 
-Custom EFI System Partition location
----------------------
-`fwupdate` 10 and later allow using an EFI system partition location
-at runtime that is different than the location compiled into the library.
+Custom EFI System Partition
+---------------------------
 
-fwupd 1.0.6 and later can take advantage of this feature by allowing
+fwupd 1.0.6 and later can use a custom EFI system partition location by allowing
```

Comment:
It is probably worthwhile to document the 3 locations that we automatically detect, `/boot/efi`, `/boot`, `/efi` in the `README.md` too.

Reply:
Fixed, thanks.